tests/dataset/test_aoi.py

```diff
@@ -261,7 +264,7 @@ def test_write_multiband_from_single_band(self) -> None:
         data = read_csv("tests/tiling/tiling_segmentation_binary-singleband_ci.csv")
         row = data[0]
         aoi = AOI(raster=row['tif'], label=row['gpkg'], split=row['split'], raster_bands_request=['R', 'G', 'B'],
-                  write_multiband=True, root_dir="data")
+                  write_dest_raster=True, root_dir="data")
         assert Path("data/SpaceNet_AOI_2_Las_Vegas-056155973080_01_P001-WV03-R-G-B.tif").is_file()
         aoi.close_raster()
```

Comment:
I would add one more assertion here:
assert rasterio.open("data/SpaceNet_AOI_2_Las_Vegas-056155973080_01_P001-WV03-R-G-B.tif").count == 3

This way we also check:
1. The resulted raster is a valid GeoTIFF
2. All three bands were written.